### ISSUE TITLE
iCloud account strangeness

### ISSUE THREAD

**Issue by strangebirds** [Ranchero-Software/NetNewsWire#2947]
I'm not sure this is the same as [#2938](https://github.com/Ranchero-Software/NetNewsWire/issues/2938
)
29 March: 6.0.1b1 - tried adding an iCloud account. Selected all feeds (click, shift-click) in "On My Mac" and dragged them to iCloud in sidebar. About halfway through, NNW crashed (and I reopened without sending the crash log - it is attached, zipped, below (strange that github doesn't allow .crash files)). On reopen, both "On My Mac" and iCloud had feeds, but not all had copied to the iCloud account.

30 March: updated to 6.0.1b2 Tried adding feeds that did not copy to iCloud and got errors (which I failed to note down - recommended making sure iCloud was enabled). Erased all feeds from iCloud, and made it inactive in preferences. Checked github for issues, decided to try repeating.
1. Made iCloud account active: **erased feeds appeared in sidebar.** After using refresh button they disappeared.
2. dragged all feeds (188 in 13 folders) from "On My Mac" to "iCloud"
3. Refreshed feeds: **operation showed 967 feeds updating rather than 376**
4. "On my Mac" feeds refreshed as expected. iCloud is…extremely slow. At >5 minutes 1 of 188 feeds shows refreshed.
5. Quit NNW. Restarted. Refresh is fine: speed is normal, no error messages

I am unsure if I did something unexpected or iCloud was being wonky or both.
Let me know if there is more info or logs, profile, etc. that would be helpful.

[NetNewsWire_2021-03-29-175216_Vanship.crash.zip](https://github.com/Ranchero-Software/NetNewsWire/files/6228262/NetNewsWire_2021-03-29-175216_Vanship.crash.zip)

**Comment by strangebirds** [comment]
Yes. I figured the initial iCloud slowness was on their end.

thanks,
Ethan Zlomke
@strangebirds
***@***.***
(Fallback email: ***@***.***)

**Comment by strangebirds** [comment]
The initial crash is the only concern on this issue. NNW behaving as expected now that iCloud has "settled".
No further slowness or crashes noted.

**Comment by vincode-io** [comment]
@brentsimmons Is that crash something you can symbolicate? I'm not having any luck with it.

**Comment by brentsimmons** [comment]
Yes — I’ll symbolicate it as soon as I get chance, and the post it here.
